.github/workflows/rust.yml

```
@@ -244,6 +246,7 @@ jobs:
     wasm_tests:
         name: Test (WASM)
         runs-on: ubuntu-latest
+        timeout-minutes: 25
```

These tests are very slow but better to wait for @lexnv knows this best

```suggestion
        timeout-minutes: 45
```